server/pkg/schema/field.go

```diff
@@ -146,12 +148,22 @@ func (f *Field) Clone() *Field {
 	}
 }
 
-func (f *Field) Validate(v *value.Value) error {
-	if f.required && v.IsEmpty() {
+// Validate the Multiple value against the Field schema
+// if its multiple it will return only the first error
+func (f *Field) Validate(vs *value.Multiple) error {
+	if f.required && (vs.IsEmpty() || vs.First().IsEmpty()) {
 		return ErrValueRequired
 	}
-	if v.IsEmpty() {
+	if vs.IsEmpty() {
 		return nil
 	}
-	return f.typeProperty.Validate(v)
+	if !f.multiple && len(vs.Values()) > 1 {
```

Comment:
```suggestion
	if !f.multiple && vs.Len() > 1 {
```

Impl `func (*value.Multiple) Len() int` method to prevent cloning a slice of values.